apps/condo/domains/notification/utils/serverSchema/index.js

```diff
@@ -20,6 +20,8 @@ const { MESSAGE_TYPES } = require('@condo/domains/notification/constants/constan
 const { DISCONNECT_USER_FROM_DEVICE_MUTATION } = require('@condo/domains/notification/gql')
 /* AUTOGENERATE MARKER <IMPORT> */
 
+const LOCALE_KEYS = Object.keys(LOCALES)
+
```

Comment:
Please, move it to 'common' domain, it's a general-purpose constant

Reply:
It's better to use something like `LOCALES[data.lang]` rather than add a new const